catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentProducerDataAccessObject.java

```diff
@@ -137,10 +148,17 @@ public void createContentItem(
       throws SourceUnavailableException, IngestException {
     LOGGER.debug("Creating content item.");
 
+    if (!eventType.equals(ENTRY_DELETE) && ingestedFile == null) {
+      if (LOGGER.isDebugEnabled()) {
+        LOGGER.debug(
+            "Ingested File was null with eventType [{}]. Doing nothing.", eventType.name());
```

Comment:
Is `eventType.name()` an expensive call? Is that why there is an `if` guard `isDebugEnabled`?

Reply:
SonarLint gives warning on conditionally calling log statements. I didn't feel strongly either way, but wanted to be consistent if we are incorporating SonarLint into the development process.